client-src/src/components/GlobalNav/GlobalNav.min.css

```diff
@@ -0,0 +1 @@
+.globalNav p{color:#fff}.title{padding-left:2rem}.separator{background-color:#b8dee4}.github:hover{color:#fff}.github:hover:hover{color:#fff}.stack{position:relative;cursor:pointer}.stack .user{font-size:1.2rem;color:#0f93a6;line-height:3.3rem}.stack:hover .circle{color:#fff}.stack .circle{font-size:2rem;color:#b8dee4;line-height:3.3rem}
```

Comment:
Looks like this snuck back in, might want to check your editor plugins to see what is building this.